container/src/LuigiContainer.svelte

```diff
@@ -5,6 +5,8 @@
   export let context;
   export let label;
   export let webcomponent;
+  export let can_mfe_initialized;
+  export let mfe_initialized_ready;
```

Comment:
What does `can_mfe_initialized` mean. Also `mfe_initialized_ready`. I have a general idea but it'd be good to have small inline comments as we might forget later on.